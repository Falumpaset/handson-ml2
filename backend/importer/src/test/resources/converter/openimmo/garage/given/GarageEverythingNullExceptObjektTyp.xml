<?xml version="1.0" ?>
<openimmo xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="">
    <uebertragung art="ONLINE" umfang="TEIL" version="1.2.7" sendersoftware="Immomio"
                  senderversion="1.0"></uebertragung>
    <anbieter>
        <anbieternr>jbbg402699</anbieternr>
        <firma>test</firma>
        <openimmo_anid>AIMIO16159437268522153127c0b5da</openimmo_anid>
        <immobilie>
            <objektkategorie>
                <nutzungsart WOHNEN="false" GEWERBE="false" ANLAGE="true"></nutzungsart>
                <vermarktungsart KAUF="false" MIETE_PACHT="true"></vermarktungsart>
                <objektart>
                    <parken parken_typ="DUPLEX"></parken>
                </objektart>
            </objektkategorie>
        </immobilie>
    </anbieter>
</openimmo>