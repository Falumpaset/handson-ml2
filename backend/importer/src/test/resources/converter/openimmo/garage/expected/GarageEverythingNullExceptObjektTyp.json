{
  "id": null,
  "type": "GARAGE",
  "externalId": null,
  "data": {
    "name": null,
    "size": null,
    "attic": false,
    "floor": null,
    "rooms": null,
    "garden": false,
    "ground": null,
    "heater": null,
    "address": null,
    "contact": null,
    "seniors": false,
    "bailment": null,
    "elevator": false,
    "flatType": null,
    "landArea": null,
    "basePrice": null,
    "bathRooms": null,
    "documents": [],
    "fireplace": false,
    "flatShare": false,
    "gardenUse": false,
    "houseType": null,
    "storeRoom": false,
    "externalId": null,
    "garageData": {
      "width": null,
      "height": null,
      "length": null,
      "freeUntil": null,
      "garageType": "DUPLEX"
    },
    "gardenArea": null,
    "objectType": "GARAGE",
    "tvSatCable": false,
    "attachments": [],
    "barrierFree": false,
    "bicycleRoom": false,
    "heatingCost": null,
    "kitchenette": false,
    "referenceId": null,
    "showAddress": false,
    "showContact": false,
    "washDryRoom": false,
    "basementSize": null,
    "availableFrom": null,
    "guestToilette": false,
    "parkingSpaces": null,
    "serviceCharge": null,
    "commercialData": null,
    "furnishingType": null,
    "numberOfFloors": null,
    "previousTenant": null,
    "totalRentGross": null,
    "temporaryLiving": null,
    "constructionYear": null,
    "historicBuilding": false,
    "numberOfBedrooms": null,
    "numberOfTerraces": null,
    "basementAvailable": null,
    "bathroomEquipment": null,
    "buildingCondition": null,
    "energyCertificate": null,
    "lastRefurbishment": null,
    "numberOfBalconies": null,
    "objectDescription": null,
    "totalParkingPrice": null,
    "balconyTerraceArea": null,
    "objectLocationText": null,
    "heatingCostIncluded": false,
    "furnishingDescription": null,
    "objectMiscellaneousText": null,
    "parkingPriceIncludedToAdditionalCosts": false
  }
}